lib/index.js

```diff
@@ -1,9 +1,10 @@
-import isEmpty from 'lodash/isEmpty';
-import platform from 'platform';
+// for some dom traverse, while(true) is useful
+/* eslint-disable no-constant-condition */
```

Comment:
It's only used twice. Just use `// eslint-disable-next-line no-constant-condition` at those two spots, rather than the whole module. (It also doesn't match the explanation about DOM traversal; the spots where it's used are for retry loops.)

Reply:
updated~